lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,2168 @@
+{
```

Comment:
<p>Tady také můžeme použít <code>raise_for_status()</code>  a nechat funkci ukončit, pokud nastane problém s komunikací s API.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='93'/>